src/backend/web/static/swagger/api_v3.json

```diff
@@ -8322,6 +8383,55 @@
           "year"
         ]
       },
+      "NotablesInsight": {
+        "type": "array",
+        "items": {
+          "type": "object",
+          "properties": {
+            "data": {
+              "type": "object",
+              "properties": {
+                "entries": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "context": {
+                        "type": "array",
+                        "description": "A list of events this team achieved the notable at. This type may change over time.",
```

Comment:
What do you mean by "change over time"? How do you see this changing?

Reply:
It may add more context - for example for WFFAs, we may want to add awardee. Not sure. It's more of a disclaimer than an intent to change